.cirrus.yml

```diff
@@ -0,0 +1,18 @@
+---
+
+test_task:
+  alias: "build_and_test"
+  name: "Build and Test"
+  container:
+    image: docker.io/library/rust
+  install_script: cargo build
+  test_script: cargo test
+
+success_task:
```

Comment:
This needs `alias: success` otherwise it will break a future `hack/get_ci_vm.sh`.